plugins/view-secret.yaml

```diff
@@ -3,19 +3,45 @@ kind: Plugin
 metadata:
   name: view-secret
 spec:
+  version: "v0.2.0"
+  homepage: https://github.com/elsesiy/kubectl-view-secret
+  shortDescription: |
+    Decode Kubernetes secrets
+  description: |+2
+    Base64 decode by key or all key/value pairs in a given secret.
+
+    $ kubectl view-secret <secret> # print secret keys
```

Comment:
nit: can you align the comments? That would read so much nicer...

Reply:
> 2\. The output is still not quite the same.

I think this is ok.

Reply:
Will do but I just noticed I'm also missing [this](https://github.com/ahmetb/kubectl-extras/blob/master/view-secret/view-secret.sh#L38) case here. Do you want me to add it as well?

Reply:
I think it would be nice. Many Secrets will only have 1 key.

Do this:
1. print these communication messages to `stderr`, print parseable output (by scripts etc) to `stdout`
2. offer a `-q`/`--quiet` to turn off communication messages like ("I'm selecting the only key").

Reply:
Done!